Cargo.toml

```diff
@@ -10,28 +10,42 @@ build = "build.rs"
 [dependencies]
 tokio = { version = "1", features = ["full"] }
 axum = { version = "0.6.0-rc.2", features = ["json"] }
+hyper = "0.14.20"
```

Comment:
It's best to not specify the 'patch' part of the version, e.g. `0.14` in this case. This applies to all dependencies, since it makes it easier for cargo to resolve compatible versions in the dependency tree and minimize duplication.

Reply:
Resolved